tools/getting-started/src/ui.tsx

```diff
@@ -148,7 +148,7 @@ const Ui: FC = () => {
               <Box marginBottom={1} />
 
               <LabelValueInput
-                label="Coveralls token"
+                label="Coveralls token [optional]"
                 inputValue={coverallsToken}
```

Comment:
You should add the `Press enter to skip` placeholder like the others.
Also, do we want to provide support setup for codecov in the CLI? 
If so, let's create a new issue I guess.

Reply:
Alright lets create an issue for an additional optional command for codecov. I'll wrap up this once since I'm already here